components/eam/src/chemistry/mozart/mo_chm_diags.F90

```diff
@@ -1037,6 +1037,7 @@ subroutine chm_diags( lchnk, ncol, vmr, mmr, rxt_rates, invariants, depvel, depf
       wrk1d(:) = 0._r8
       do i = 1,ncol
          ! from top to surface
+         j = 2 !kzm added to initialize j
```

Comment:
Will j=1 be safer? I am not sure why this can cause issue when debug built. A bit more information will be helpful.

Reply:
@tangq we can use j=1